Article 120 random variables

OmniaKahla opened the pull request:
## Attributes

<!-- Please use one of these items per category by removing the comment tag at the beginning and end of the line. -->

#### Length of text ####
 :page_with_curl: short 
<!-- :book:  long -->

####  Difficulty to read ####
 :mouse:  light 
<!-- :elephant:  heavy -->

####  Difficulty to understand #### 
 :school_satchel:  easy 
<!-- :mortar_board:  difficult -->

####  Type of text #### 
<!-- :laughing:  funny -->
 :bowtie:  informative 
 :sunglasses:  catchy 

####  Type of header #### 
<!-- :santa:  funny --> 
 :bulb:  informative 
<!-- :boom:  catchy -->

Fjolyo commented:
# Plenum Suggestions:
- Right part of image is too confusing
- Don't refer to Monte Carlo and density functions
- Skip "What is a random process?"
- Maybe switch order of paragraphs to start with the algebra part

OmniaKahla commented:
All changes have been done as follow:
- The right part of the image has been removed.
-  Removed the Monte Carlo and density function.
-  I kept the question, as it seems we are talking with the audience.
-  Added the difference between the Algebra variables and the random variables to the first paragraph.

Teifun2 commented:
> * The right part of the image has been removed.

Looks better than before.

> * Removed the Monte Carlo and density function.

Looks good.

> * I kept the question, as it seems we are talking with the audience.

We are still not a fan of it, but if you want, it can stay in.

> * Added the difference between the Algebra variables and the random variables to the first paragraph.

The explanation seems a little short, but we think its sufficient.


There are still some grammatical errors that need to be fixed before it can go to testing.

OmniaKahla commented:
> There are still some grammatical errors that need to be fixed before it can go to testing.
Could you please point them out, as I have checked several times and cannot find any?
Thank you.

MoiraZuber commented:
Added the image, no significant changes.

OmniaKahla commented:
Removed the question in the 1st paragraph.
